### PR TITLE
ProfilePage static, and minor improvement on loginPage

### PR DESCRIPTION
**Pull request by BBriset**

### CODE REVIEW

File: src/pages/Profile/style.jsx [hunk 1]
```diff
@@ -87,6 +88,59 @@ const SProfilPage = styled.section`
...
+  }
+  button{
+  margin-right :2vw;
+}
+  }
+  img{
+    width:60vw;
+    margin-top:-0.75vh;
+      }
+  .UserCard {
```
Comment (HeloiseFerla) [inline src/pages/Profile/style.jsx]: CamelCase ?

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Profile/index.jsx [hunk 2]
```diff
@@ -1,35 +1,63 @@
...
 import SProfilPage from './style';
-import apple from '../../assets/Img/apple.png';
-import samsung from '../../assets/Img/samsung.png';
-import applePay from '../../assets/Img/Apple_Pay_logo.svg';
-import cb from '../../assets/Img/logo-cb.svg';
-import larrondi from '../../assets/Img/LARRONDI.svg';
-import logo from '../../assets/Img/easyApp.png';
-import card from '../../assets/Img/easyCard.png';
 
 const solde = '37,64€';
```
Comment (Hacene010) [inline src/pages/Profile/index.jsx]: you could put your data into an object : )

File: src/pages/Profile/style.jsx [hunk 3]
```diff
@@ -63,7 +65,7 @@ const SProfilPage = styled.section`
   .logo {
     display: flex;
     flex-direction: row;
-    width: 80vw;
+
```
Comment (Hacene010) [inline src/pages/Profile/style.jsx]: empty lines

**Review by Hacene010 — CHANGES_REQUESTED** [review]

File: src/pages/Profile/style.jsx [hunk 4]
```diff
@@ -87,6 +87,59 @@ const SProfilPage = styled.section`
...
+    align-items: center;
+    width: 90vw;
+      img {
+    height: 60px;
+    width: auto;
+    margin-top:0;
+  }
+  button{
+  margin-right :2vw;
+}
```
Comment (HeloiseFerla) [inline src/pages/Profile/style.jsx]: indentation

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**